components/ModEditFacilityOrProfessionalTopbar.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <div class="flex flex-row justify-between w-full">
```

Comment:
nit: flex-row is redundant

```suggestion
    <div class="flex justify-between w-full">
```